webpack/common.js

```diff
@@ -167,24 +167,34 @@ const noAMDParserRule = { parser: { amd: false } };
 
 exports.noAMDParserRule = noAMDParserRule;
 
-const svgRule = {
-	test: /\.svg$/,
-	use: [
-		{
-			loader: '@svgr/webpack',
-			options: {
-				// strip width & height to allow manual override using props
-				dimensions: false,
+const svgRules = [
```

Comment:
Do we need to change it to be an array? We still have just one svg rule, don't we?

Reply:
Good point, these changes are no longer needed as we've gone back to one rule object using oneOf